src/custompresets.asm

```diff
@@ -466,6 +466,15 @@ preset_scroll_fixes:
     ; -----------------------------------------------
     ; Specialized Fixes (Category and Custom Presets)
     ; -----------------------------------------------
+  .specialized_parlor
+    LDY !SAMUS_Y : CPY #$00D0    ; no fix if Ypos > 208
+    BPL .specialdone
+    LDY !SAMUS_X : CPY #$0175    ; no fix if Xpos > 373
+    BPL .specialdone
+    %a16() : LDA #$00FF
+    STA $7F05C0 : STA $7F05C2
```

Comment:
Some of the later presets that don't have screw attack would benefit from all 6 tiles being removed.
    STA $7F0480 : STA $7F0482
    STA $7F0520 : STA $7F0522

Reply:
Yeah I would like to, not sure if I'm going to get to it.

I want to release this update soon, but feels like half of what I worked on is only half-done, still more work to do.